textile/fabric_pretreatment/doctype/pretreatment_order/pretreatment_order.py

```diff
@@ -281,7 +281,7 @@ def create_ready_fabric_bom(self, ignore_version=True, ignore_feed=True):
 		bom_doc = self.make_ready_fabric_bom()
 		bom_doc.flags.ignore_version = ignore_version
 		bom_doc.flags.ignore_feed = ignore_feed
-		bom_doc.flags.ignore_permissions = True
+		bom_doc.flags.ignore_permissions = ignore_permissions
```

Comment:
This should always be True. Check permissions separately before creating document